### PR TITLE
Added conditional element documentation

### PR DESCRIPTION
**Pull request by yosilevy**
Will add an example later on.

PR:
https://github.com/home-assistant/home-assistant-polymer/pull/2865

### CODE REVIEW

File: source/_lovelace/picture-elements.markdown [hunk 1]
```diff
@@ -397,6 +397,38 @@ style:
...
+    state:
+      required: false
+      description: Entity state is equal to this value.*
+      type: string
+    state_not:
+      required: false
+      description: Entity state is unequal to this value.*
+      type: string
+elements:
+  required: false
```
Comment (balloob) [inline source/_lovelace/picture-elements.markdown]: Should be required ?

Comment (yosilevy) [reply source/_lovelace/picture-elements.markdown]: Yes. Bad copy/paste. Fixed

**Review by frenck — CHANGES_REQUESTED** [review]
This PR has a parent PR in our codebase, and therefore, has a dependency on the next release of Home Assistant. Please change the target branch of this PR to `next` and rebase the PR if needed.

Thanks 👍

**Comment by yosilevy** [comment]
I'm really not sure how to do this. I committed these changes via the git GUI since they're minor. Is it possible to do what you asked via the GUI or should I pull the PR and do it again on the next branch?

**Comment by frenck** [comment]
Rebased.

**Pull request merged**